demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -1381,3 +1367,58 @@ def test_safe_write_unicode_to_non_unicode(
     result = get_file(dest, suffix)
     assert set(result.keys()) == set(fields)
     assert set(result.values()) == {SENTENCE_WITH_UMLAUTS}
+
+
+@pytest.fixture
+def demisto_client_configure(mocker):
+    mocker.patch(
+        "demisto_sdk.commands.upload.uploader.get_demisto_version",
+        return_value=parse("6.0.0"),
+    )
+    mocker.patch(
+        "demisto_sdk.commands.common.content.objects.pack_objects.integration.integration.get_demisto_version",
+        return_value=parse("6.0.0"),
+    )
+    mocker.patch(
+        "demisto_sdk.commands.common.content.objects.pack_objects.script.script.get_demisto_version",
+        return_value=parse("6.0.0"),
+    )
+    mocker.patch("builtins.print")
+
+
+def util_load_file_bytes(path) -> bytes:
+    with io.open(path, mode="rb") as f:
+        return f.read()
+
+
+def test_find_uuids_in_content_item():
+    """
+    Given: a mock tar file download_tar.tar
```

Comment:
can we generate this tar file dynamically using the `repo` fixture, instead of saving another file to the repo?

Reply:
I can create mock objects holding just the name and id and references to other content items UUIDs
But its a task on its own i think. and packing it to a tar the same as the server does, it might lead to a different file. Anyway I feel like it will take me allot of time just to save 24Kb the size of the tar in repo.
So I prefer to leave it as is for the moment.